api/app/reviews/tests.py

```diff
@@ -1285,3 +1288,204 @@ def test_total_number_of_pages_when_zero(self):
 
         self.assertEqual(data['total_pages'], 0)
     
+
+class ReferenceReviewRequest(ApiTestCase):
+    def static_seed_data(self):
+    # Set up organisation and user
+        self.add_organisation('Deep Learning Indaba 2019', 'blah.png', 'blah_big.png')
+        self.add_organisation('Deep Learning Indaba 2020', 'blah.png', 'blah_big.png')
+        user_categories = [
+            UserCategory('Honours'),
+            UserCategory('Student'),
+            UserCategory('MSc'),
+            UserCategory('PhD')
+        ]
+        db.session.add_all(user_categories)
+        db.session.commit()
+
+
+
+        countries = [
+            Country('Egypt'),
+            Country('Botswana'),
+            Country('Namibia'),
+            Country('Zimbabwe'),
+            Country('Mozambique'),
+            Country('Ghana'),
+            Country('Nigeria')
+        ]
+        db.session.add_all(countries)
+        db.session.commit()

```

Comment:
The organisations, countries and user categories above this line aren't needed because they now get set up by default in the ApiTestCase setUp method :)